File: 0026 删除排序数组中的重复项/main.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        count=0
        idx=1
        length = len(nums)
        while count < length-1:
            count +=1
            if(nums[idx]==nums[idx-1]):
                nums.pop(idx)
            else:
                idx+=1
        return len(nums)

File: 0026 删除排序数组中的重复项/main.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int idx=0;
        for (int i=0;i<nums.size();i++){
            if(i==0 || nums[i] != nums[i-1]){
                if(idx!=i)
                    nums[idx]=nums[i];
                idx++;
            }
        }
        return idx;
    }
};